3rdParty/StormLib/src/FileStream.cpp

```diff
@@ -18,8 +18,8 @@
 #include "StormCommon.h"
 #include "FileStream.h"
 
-#ifdef _MSC_VER
-#pragma comment(lib, "wininet.lib")             // Internet functions for HTTP stream
+#if 1//def _MSC_VER

```

Comment:
@marty28 the changes to StormLib are a bit rough handed, could you try and clean it up a bit so that it doesn't break other platforms?

Reply:
yeah sorry, I would have kept things a bit nicer but at the time I did not expect it to get merged back.Thanks for doing the cleanup :)

Reply:
I think there are still some unessesery ifdefs left in there that you could maybe clean up :)